debezium-core/src/main/java/io/debezium/transforms/debeziumtimestampcoverter/README.md

```diff
@@ -0,0 +1,51 @@
+SMT - Debezium Timestamp Converter:(Extended Version of SMT timestampConverter from connect, supporting debezium or others).
+Also works for any kind of nested org.apache.kafka.connect.data.Struct data.
+
+1. Connect config's:
+	Would split the usage into two parts,
+
+	a.Straight fields, like ts_ms in debezium.
+
+		field – the field name (optional, can be left out in case of primitive data)
+		target.type – desired type (i.e. string, long, Date, Time, Timestamp)
+		format – in case converting to or from a string, a SimpleDateFormat-compatible format string
+
+		example:
+		"transforms":"TimestampConv"
+		"transforms.TimestampConv.type":"org.telmate.SMT.TimestampConverter$Value"
+		"transforms.TimestampConv.field":"ts_ms"
+		"transforms.TimestampConvtarget.type":"Timestamp"
+		"transforms.TimestampConv.timestamp.format":"yyyy-MM-dd hh:mm:ss"
+		"transforms.TimestampConv.date.format":"yyyy-MM-dd"
+
+
+	b.Nested fields, like after or before.
+
+		struct.field - the field which is of Struct type(has nested data)
+		field.type - is a comma seperated string, providing an option to add multiple converters.
+					<Type of Timestamp Object> -> <To Target type>
+					can add multiple type converters, <Type of Timestamp Object> -> <To Target type>,<Type of Timestamp Object> -> <To Target type>
+
+
+		example:
+		"transforms.TimestampConv.struct.field":"after"
+		"transforms.TimestampConv.field.type":"io.debezium.time.Timestamp->string,io.debezium.time.Date->string"
+
+	c. Can use the combination of both (a) and (b).
+	
+
+Default Timestamp/Date Format
```

Comment:
Can one have different formats for different fields? It probably should be tied to the fields selected as suggested above.